src/nih_wayfinding/app/index.html

```diff
@@ -92,7 +92,6 @@ <h2 class="navbar-actionheader" ng-cloak ui-sref="profiles">{{ navbar.config.tit
     <script src="bower_components/angular-leaflet-directive/dist/angular-leaflet-directive.js"></script>
     <script src="bower_components/Leaflet.encoded/Polyline.encoded.js"></script>
     <script src="bower_components/turf/turf.min.js"></script>
-    <script src="bower_components/leaflet-plugins/layer/tile/Google.js"></script>
```

Comment:
@ddohler  this is actually the one that needs to stay in right?

Probably still need finer controls on the exclude section...?